### PR TITLE
Get server to serve static files

### PR DESCRIPTION
**Pull request by thomasmunduchira**
Integration tests added for static file serving as well. Tests are split up since now only unit tests are going to be tested on Travis. Integration tests should be run locally before code is pushed.

### CODE REVIEW

File: src/controller/routes.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from flask import abort, Blueprint, send_from_directory
+import os
+
+routes = Blueprint('routes', __name__)
```
Comment (klanmiko) [inline src/controller/routes.py]: can we rename this blueprint and file as static_routes or serve_static

Comment (thomasmunduchira) [reply src/controller/routes.py]: Yeah

Comment (thomasmunduchira) [reply src/controller/routes.py]: Wait can we just use this file for all the routes, even the ones for review?

Comment (klanmiko) [reply src/controller/routes.py]: the review route has to do auth, row locking, and like two levels of db query, I need to put it somewhere would be nice to split up the two route modules or else we can make a controller class in a separate file with a single route module

Comment (thomasmunduchira) [reply src/controller/routes.py]: Okay I'll split it up

**Pull request merged**